ts/commands/ReportCommands.ts

```diff
@@ -19,6 +19,22 @@ class ReportCommands {
         const config = configService.mergeConfig(options);
         reportService.list(config, logger);
       });
+
+    (caporal as any)
+      .command('report ec2cost', 'Retrieve ec2 cost data grouped and aggregated.')
+      .option('--username', 'Metricly Username')
+      .option('--password', 'Metricly Password')
+      .option('--profile', 'Metricly profile', /.*/, 'default')
+      .option('--format', 'format options: text, json', ['text', 'json'], 'text')
+      .option('--timerange', 'relative timerange options: latestDay, monthToDate, lastMonth', ['latestDay', 'monthToDate', 'lastMonth'], 'latestDay')
```

Comment:
based on some other reports, to be consistent I think I should rename to `period`